Decisions.Equifax/EquifaxRequest.cs

```diff
@@ -0,0 +1,694 @@
+﻿using System.Runtime.Serialization;
+using DecisionsFramework.Design.ConfigurationStorage.Attributes;
+using Newtonsoft.Json;
+
+namespace Decisions.Equifax
+{
+    public class EquifaxRequest
+    {
+
+        public partial class EquifaxUsConsumerCreditReport
+        {
+            [JsonProperty("identifier")] public string Identifier { get; set; }
+
+            [JsonProperty("customerReferenceNumber")]
+            public string CustomerReferenceNumber { get; set; }
+
+            [JsonProperty("customerNumber")] public string CustomerNumber { get; set; }
```

Comment:
Rider didn't do any favors here. Can we break these back out to two lines each so they're more readable?

Reply:
no worries, I had fixed those before, guess Rider put them back. done